### DISCUSSION TITLE
Is it possible to get console error message if sending messages fails?

### DISCUSSION THREAD

**Discussion by rpeveng** [osiegmar/logback-gelf#88] [Q&A] [answered]
Test case: Configure GELF TCP appender with a port where no graylog input listens.

Graylog then adds statuses to Logback's status manager but without explicitly listening to them nothing is visible in the console output.

Can something be enabled in configuration to get a console output if TCP/UDP connection cannot be established?

**Comment by osiegmar** [comment, marked as answer]
In order to see problems/errors within the log system itself, you can use logbacks [Status data](https://logback.qos.ch/manual/configuration.html#dumpingStatusData) feature. In the simplest form you add the `debug` attribute to your `<configuration>` root element of your logback configuration: `<configuration debug="true">`. That causes internal errors to be logged to the console.
